tools/loadtesting/terraform/rds.tf

```diff
@@ -0,0 +1,76 @@
+resource "random_password" "database_password" {
+  length  = 16
+  special = false
+}
+
+resource "random_pet" "db_secret_postfix" {
+  length = 1
+}
+
+resource "aws_secretsmanager_secret" "database_password_secret" {
+  name = "/fleet/database/password/master-2-${random_pet.db_secret_postfix.id}"
+}
+
+resource "aws_secretsmanager_secret_version" "database_password_secret_version" {
+  secret_id     = aws_secretsmanager_secret.database_password_secret.id
+  secret_string = random_password.database_password.result
+}
+
+module "aurora_mysql" {
+  source  = "terraform-aws-modules/rds-aurora/aws"
+  version = "5.3.0"
+
+  name                  = "${local.name}-mysql-iam"
+  engine                = "aurora-mysql"
+  engine_version        = "5.7.mysql_aurora.2.10.0"
+  instance_type         = "db.r5.2xlarge"
+  instance_type_replica = "db.r5.2xlarge"
+
+  iam_database_authentication_enabled = true
+  storage_encrypted                   = true
+  username                            = "fleet"
+  password                            = random_password.database_password.result
+  create_random_password              = false
+  database_name                       = "fleet"
+  enable_http_endpoint                = false
```

Comment:
we might want to set this to true so we can debug db from AWS console without having to stand up bastion or punch holes in VPC